### PR TITLE
Add more bookmark icon variant

### PR DESCRIPTION
**Pull request by siarie**
**Preview**
Preview

100%
bookmark-add

### CODE REVIEW

**Review by ericfennis — CHANGES_REQUESTED** [review]
@siarie Awesome icon!
Can you rename it to `bookmark-plus`?
Then the naming matches with the other "plus" icons.
As optional you could apply some extra tags in "tags.json", like the word "add" so you can find it more easily on the [lucide.dev](https://lucide.dev) website.

**Comment by siarie** [comment]
Hi @ericfennis, I've rename it to `bookmark-plus`.

File: icons/bookmark-add.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M19 21l-7-5-7 5V5a2 2 0 012-2h10a2 2 0 012 2z" />
+  <line x1="12" x2="12" y1="7" y2="12" />
+  <line x1="14.5" x2="9.5" y1="9.5" y2="9.5" />
```
Comment (ericfennis) [inline icons/bookmark-add.svg]:
```suggestion
  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z"/>
  <line x1="12" x2="12" y1="7" y2="13" />
  <line x1="15" x2="9" y1="10" y2="10" />
```

**Review by ericfennis — COMMENTED** [review]
@siarie One thing, I see you used half pixels in the `plus` part.
That will result in blurry pixels on low res devices.
I think it is better if we slightly increase the size of the plus so we are on rounded pixels. But we need to "blunting" the point a bit.
We need to change this as wel in the `bookmark` icon.

Left is yours, new one on the right.
image

File: icons/bookmark-add.svg (hunk 1 above)
Comment (siarie) [reply icons/bookmark-add.svg]: Well actually I copied from the bookmark icon and just added the plus icon. Do I need to change the `bookmark` too?

Comment (ericfennis) [reply icons/bookmark-add.svg]: Yes if you can do 😃, that would be great.
And maybe you can add a `bookmark-minus` as wel.

Comment (siarie) [reply icons/bookmark-add.svg]: Sure.

Comment (siarie) [reply icons/bookmark-add.svg]: ✅ Done.

File: icons/bookmark.svg [hunk 2]
```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M19 21l-7-5-7 5V5a2 2 0 012-2h10a2 2 0 012 2z" />
+  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z"/>
```
Comment (ericfennis) [inline icons/bookmark.svg:12]:
```suggestion
  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z" />
```

File: icons/bookmark-minus.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z"/>
+  <line x1="15" x2="9" y1="10" y2="10"/>
```
Comment (ericfennis) [inline icons/bookmark-minus.svg:13]:
```suggestion
  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z" />
  <line x1="15" x2="9" y1="10" y2="10" />
```

File: icons/bookmark-plus.svg [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z"/>
+  <line x1="12" x2="12" y1="7" y2="13"/>
+  <line x1="15" x2="9" y1="10" y2="10"/>
```
Comment (ericfennis) [inline icons/bookmark-plus.svg:14]:
```suggestion
  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z" />
  <line x1="12" x2="12" y1="7" y2="13" />
  <line x1="15" x2="9" y1="10" y2="10" />
```

**Review by ericfennis — COMMENTED** [review]
@siarie Awesome great work. One last small code style thingy.

**Review by ericfennis — APPROVED** [review]

**Comment by ericfennis** [comment]
@siarie Great work, thanks for the new icons!

**Comment by siarie** [comment]
You're welcome... 😄

**Pull request merged**